Formula/clozure-cl.rb

```diff
@@ -0,0 +1,64 @@
+class ClozureCl < Formula
+  desc "Common Lisp implementation with a long history"
+  homepage "https://ccl.clozure.com"
+  url "https://github.com/Clozure/ccl/archive/v1.11.5.tar.gz"
+  sha256 "07c7e05c1d50ccbf1f7602a1d436b6d6da5dcda7656b89b74daa4cf833fc7929"
+  head "https://github.com/Clozure/ccl.git"
+
+  depends_on :xcode => :build
+
+  conflicts_with "cclive", :because => "both install a ccl binary"
+
+  resource "bootstrap" do
+    url "https://github.com/Clozure/ccl/releases/download/v1.11.5/ccl-1.11.5-darwinx86.tar.gz"
+    sha256 "5adbea3d8b4a2e29af30d141f781c6613844f468c0ccfa11bae908c3e9641939"
+  end
+
+  def install
+    # Unfortunately, this program is a bit finicky about which architecture, or
+    # specifically which bitness the machine is. So that's what all this
+    # business is with `MacOS.prefer_64_bit?`
+
+    arch_64 = MacOS.prefer_64_bit?
```

Comment:
macOS is 64-bit only now, which makes the formula a lot simpler:

```
    tmpdir = Pathname.new(Dir.mktmpdir)
    tmpdir.install resource("bootstrap")
    buildpath.install tmpdir/"dx86cl64.image"
    buildpath.install tmpdir/"darwin-x86-headers64"
    cd "lisp-kernel/darwinx8664" do
      system "make"
    end

    ENV["CCL_DEFAULT_DIRECTORY"] = buildpath
    system "./dx86cl64", "-n", "-l", "lib/x8664env.lisp",
                         "-e", "(ccl:xload-level-0)",
                         "-e", "(ccl:compile-ccl)",
                         "-e", "(quit)"
    system "echo \"(ccl:save-application \\\"dx86cl64.image\\\")\\n(quit)\" | ./dx86cl64 -n --image-name x86-boot64.image"

    prefix.install "doc/README"
    doc.install Dir["doc/*"]
    libexec.install Dir["*"]
    bin.install Dir["#{libexec}/scripts/ccl64"]
    bin.env_script_all_files(libexec/"bin", :CCL_DEFAULT_DIRECTORY => libexec)
```

Reply:
@fxcoudert No, it isn't. It just warns you when you run 32-bit applications. And even that only happens in recent versions of macOS. As I understand, homebrew has a significant amount users on older versions. Perhaps 10.14 *will* completely forbid 32-bit applications, but it has not happened yet. And even when it does, many users will not upgrade.

Reply:
Apple hasn't manufactured a 32-bit-only Mac since 2005 and Homebrew only provides best-effort support for versions below 10.11, so I think we're safe in shipping only the 64-bit version here.

Reply:
If things build out of the box for 32-bit, I'm happy to keep it. But let's not bend over backward and make formulas uglier for it.

Reply:
@woodruffw @fxcoudert Ok, ok, you guys win. I removed the code for 32-bit builds.